org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/MavenTargetLocationWizard.java

```diff
@@ -90,6 +92,9 @@ public MavenTargetLocationWizard(MavenTargetLocation targetLocation) {
 				targetLocation == null ? Messages.MavenTargetLocationWizard_1 : Messages.MavenTargetLocationWizard_2) {
 
 			private Link editInstructionsButton;
+			private Label scopeDescriptionLabel;
+
+			private Label includeLabel;
```

Comment:
Shouldn't this be named `depthDescriptionLabel`?

Reply:
Maybe yes, anyways as it is never presented to the users its just up to us to choose the name, we still can refactor this later.